packages/nimble-components/src/label-provider/rich-text/index.ts

```diff
@@ -39,6 +41,9 @@ export class LabelProviderRichText
     @attr({ attribute: 'toggle-numbered-list' })
     public toggleNumberedList: string | undefined;
 
+    @attr({ attribute: 'mention-user-label' })
+    public mentionUserLabel: string | undefined;
```

Comment:
User mentions are not applicable to all RichText elements. User mentions are like a plugin. Theoretically there could be many types of mentions. Some defined outside nimble. They shouldn't all be added to the label provider of rich text.

Lets instead make this an attribute on the rich-text-mention base class called `@observable buttonLabel?:string;`.

Reply:
Updated